.github/workflows/smalltalk-ci.yml

```diff
@@ -23,4 +23,4 @@ jobs:
       shell: bash
       timeout-minutes: 15
       env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
```

Comment:
I have the feeling, that this change is not intended

Reply:
This change is from a commit from @fniephaus which was added when I merged from development

Reply:
Or is it? Wait I need to check